WebApiCoreSeed.WebApi/Middleware/AuthorizationMiddleware.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace WebApiCoreSeed.WebApi.Middleware
+{
+    /// <summary>
+    /// Middleware to handle global authorization policies
+    /// </summary>
+    public class AuthorizationMiddleware
```

Comment:
This middleware is a bit.. polemic. It's custom-made to be use on BetterCare's Integration Tests, so users created for integration tests cannot be used to perform requests against the API outside the test server.

It is tightly coupled to BetterCare in this line:

```C#
var isTestUser = context.User.HasClaim(c => c.Type == "groups" && c.Value == "IntegrationTestUsers");
```

Because BetterCare integration tests users are created within that group.

I'm not sure about including this middleware in the seed, but if we do, then we might want to add a couple more comments.